web/ui/mantine-ui/src/components/NotificationsIcon.tsx

```diff
@@ -45,6 +45,7 @@ const NotificationsIcon = () => {
                       <IconAlertTriangle color="red" size={20} />
                       <Stack style={{ maxWidth: 250 }} gap={0}>
                         <Text size="sm" fw={500}>{notification.text}</Text>
+                        <>{notification.details ? <Text size="sm">{notification.details}</Text> : null }</>
```

Comment:
For "show either A or nothing" (vs. "show either A or B") you can just use `&&` in React, since falsy nodes are automatically ignored (https://legacy.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator):

```suggestion
                        <>{notification.details && <Text size="sm">{notification.details}</Text>}</>
```